src/backend/runners.ts

```diff
@@ -0,0 +1,58 @@
+import { GameManager, LibraryManager } from 'common/types/game_manager'
+
+import * as SideloadGameManager from 'backend/storeManagers/sideload/games'
+import * as SideloadLibraryManager from 'backend/storeManagers/sideload/library'
+import * as GOGGameManager from 'backend/storeManagers/gog/games'
+import * as GOGLibraryManager from 'backend/storeManagers/gog/library'
+import * as LegendaryGameManager from 'backend/storeManagers/legendary/games'
+import * as LegendaryLibraryManager from 'backend/storeManagers/legendary/library'
+import * as NileGameManager from 'backend/storeManagers/nile/games'
+import * as NileLibraryManager from 'backend/storeManagers/nile/library'
+
+interface StoreBackendConfig {
+  runner: 'legendary' | 'gog' | 'nile' | 'sideload'
+}
+
+export const runnerMap: {
+  [key in StoreBackendConfig['runner']]: {
+    store: string
```

Comment:
Maybe rename this to `storeName`/`storeId`? `store` alone doesn't immediately make it clear that it's a name (one could also imagine the store URL)

Reply:
Funny, I thought about this change as well but that. Nah it was storeMap before I'll just keep it as is. But updated it to storeId :)